File: Bridge/cpp/TimeImp.cpp
#include "TimeImp.h"

TimeImp::TimeImp(int hr, int min) {
	hr_ = hr;
	min_ = min;
}

void TimeImp::tell() {
	cout << "time is " << setw(2) << setfill('0') << hr_ << min_ << endl;
}

File: Bridge/cpp/java/src/dp/bridge/time/TimeImp.java
package dp.bridge.time;

public class TimeImp {
	protected int hr, min;
	
	public TimeImp(int hr, int min){
		this.hr = hr;
		this.min = min;
	}
	
	public void tell(){
		System.out.println("time is " + String.format("%1$02d%2$02d", this.hr, this.min));
	}
}